web/cypress/integration/user/login.spec.js

```diff
@@ -14,26 +14,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/* eslint-disable no-undef */
 
-export const BASE_URL = `http://localhost:${process.env.PORT || 8000}`;
+context('Login Test', () => {
+  it('login failed with empty username and password', () => {
+    cy.visit('/user/Login');
+    cy.contains('Login').click();
+    cy.get('.ant-form-item-explain').should('contain', 'Please input username');
+    cy.get('.ant-form-item-explain').should('contain', 'Please input password');
+  });
```

Comment:
How about login in with invalid credentials?

Reply:
yep, i will add this test case.